08_files_lab/03.py

```diff
@@ -1,3 +1,9 @@
-# Given a CSV file with numbers in the second column - 
-# print the sum of the numbers.
-#
+""" add to numbers from csv file"""
+import os,sys
+import csv
+sum = 0

```

Comment:
It's best to select names that don't collide with built-in python functions, i.e. `second_row_sum`, while longer, is better than `sum`